spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -62,7 +62,7 @@
         <dependency>
             <groupId>org.springframework.integration</groupId>
             <artifactId>spring-integration-core</artifactId>
-            <scope>test</scope>
+            <optional>true</optional>
```

Comment:
If we don't rely on Spring Integration anymore, do we still need this dependency here?

Reply:
Looks like it's needed in the tests. Would you prefer to have `scope=test`?

Reply:
Correct. `test` is better then.